### ISSUE TITLE
NICE LOGGER

### ISSUE THREAD

**Issue by Francesco2426** [cracks1/Hazard-Nuker#1]
lol skid

**Comment by NobodyWouldEverUseThis7** [comment]
fr. what a massive skid lmao. and he didnt even use a good obfuscator

**Comment by Resisst** [comment]
noot a graber i can vouch!aa!!!!!1!1!

**Comment by mastercodeon314** [comment]
This is copy pasted code from someone else who actually developed these tools, hazard nuker and hazard token grabber was ripped off of the original developers

**Comment by ClaraCrazy** [comment]
"Repository owner deleted a comment from NobodyWouldEverUseThis7 9 days ago"
*honk honk* 🤡

**Comment by mastercodeon314** [comment]
> > This is copy pasted code from someone else who actually developed these tools, hazard nuker and hazard token grabber was ripped off of the original developers
> 
> his first commits were just an actual logger

Ah, so everything else after I guess is skidded

**Comment by ClaraCrazy** [comment]
yup. obfuscated with a shitty open-source obfuscation too. lmao. its still not clean btw

**Comment by NobodyWouldEverUseThis7** [comment]
Absolute clown moment

**Comment by mastercodeon314** [comment]
> yup. obfuscated with a shitty open-source obfuscation too. lmao. its still not clean btw

Oooo, what obber was used?

**Comment by abcd8** [comment]
Grabber
